From ea058c66f2b48a61605dbf2ec8c3fce20b9ff1e6 Mon Sep 17 00:00:00 2001
From: Masami Ichikawa <masami256@gmail.com>
Date: Thu, 10 Jul 2025 21:09:44 +0900
Subject: [PATCH] kconfigs: Enable debug info options to qemu.conf

Enable debug info options to be able to get source line information.

Signed-off-by: Masami Ichikawa <masami256@gmail.com>
---
 kconfigs/qemu.conf | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/kconfigs/qemu.conf b/kconfigs/qemu.conf
index bd1d256..d3463d5 100644
--- a/kconfigs/qemu.conf
+++ b/kconfigs/qemu.conf
@@ -16,3 +16,7 @@ CONFIG_ARM_FFA_TRANSPORT=y
 CONFIG_TCG_TPM=y
 CONFIG_TCG_FTPM_TEE=y
 CONFIG_IPV6=y
+CONFIG_DEBUG_INFO=y
+CONFIG_DEBUG_INFO_DWARF_TOOLCHAIN_DEFAULT=y
+CONFIG_DEBUG_INFO_REDUCED is not set
+
-- 
2.43.0

